.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# You can apply for codeownership on our discord server
+# In this thread: https://discord.com/channels/881118111967883295/904803065293578260/907030924510781460
+# Please also ping Epic and TM (IDs: 397745647723216898 & 444548735075352576)
+*       @TAG-Epic
+
+/assets/ @DenverCoder1
+/docs/ @VincentRPS @ooliver1
+/examples/ @VincentRPS @ooliver1
+/.github/CODEOWNERS @veni-vidi-code 
+/nextcord/ext/tasks/ @veni-vidi-code 
+
+# Voice Rec
+/nextcord/ext/voicerecording/ @veni-vidi-code 
+nextcord/voice_client.py @veni-vidi-code 
+nextcord/opus.py @veni-vidi-code
+examples/receive_vc_audio.py @veni-vidi-code
+
+# Slash Commands
```

Comment:
same with what epic said this shouldn't be added(or at least commented) until it is merged